### PR TITLE
Release 0.11.2

### PR DESCRIPTION
**Pull request by PokhodenkoSA**
- [x] #653 
- [x] #678 

On finish: 
Merge or rebase to release0.11.

### CODE REVIEW

**Comment by PokhodenkoSA** [comment]
Pushed tag 0.11.2dev0. Will build package and use for numba-dppy testing

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.11.2] - 11/xx/2021
+
+## Fixed
+- Fix/gh #649 transpose (#653)
```
Comment (oleksandr-pavlyk) [inline CHANGELOG.md:10]:
```suggestion
- Fixed issue #649 about incorrect behavior of `.T` method on sliced arrays (#653)
```

**Comment by PokhodenkoSA** [comment]
Pushed tag 0.11.2rc1

**Comment by PokhodenkoSA** [comment]
Merged to fold/2021

**Pull request merged**